iceoryx_hoofs/test/moduletests/test_posix_posix_call.cpp

```diff
@@ -69,13 +70,20 @@ TEST_F(PosixCall_test, CallingFunctionWithSuccessReturnValue_GoodCase)
         })
         .or_else([](auto&) { EXPECT_TRUE(false); });
 
-    EXPECT_TRUE(internal::GetCapturedStderr().empty());
+    if (iox::testing::TestingLogger::doesLoggerSupportLogLevel(iox::log::LogLevel::ERROR))
+    {
+        auto logMessages = iox::testing::TestingLogger::getLogMessages();
+        ASSERT_EQ(logMessages.size(), 0);
+    }
+    else
+    {
+        FAIL() << "TestingLogger must support log level: ERROR";
+    }
```

Comment:
Please remove this. The log message can completely compiled out of the binary when the minimal log level is set to off. In order to not fail the test, the check should only be done when at least in potentially there would be a log message.